crowdin_api/api_resources/string_comments/resource.py

```diff
@@ -60,6 +60,7 @@ def list_string_comments(
     def add_string_comment(
         self,
         projectId: int,
+        stringId: int,
```

Comment:
Let's change the order of the parameters. The `stringId` should be placed after the `text` parameter. Just to comply with the order of the parameters as in Docs.